numcosmo/perturbations/nc_hipert_adiab.c

```diff
@@ -157,113 +165,84 @@ _nc_hipert_adiab_finalize (GObject *object)
+static gdouble
+_nc_hipert_adiab_eval_m (NcmCSQ1D *csq1d, NcmModel *model, const gdouble t, const gdouble k)
+{
+  return nc_hipert_iadiab_eval_m (NC_HIPERT_IADIAB (model), t, k);
 }
 
 static gdouble 
-_nc_hipert_adiab_eval_mnu (NcmHOAA *hoaa, NcmModel *model, const gdouble t, const gdouble k)
+_nc_hipert_adiab_eval_mnu (NcmCSQ1D *csq1d, NcmModel *model, const gdouble t, const gdouble k)
 {
   return nc_hipert_iadiab_eval_mnu (NC_HIPERT_IADIAB (model), t, k);  
 }
 
 static gdouble 
-_nc_hipert_adiab_eval_nu (NcmHOAA *hoaa, NcmModel *model, const gdouble t, const gdouble k)
+_nc_hipert_adiab_eval_nu (NcmCSQ1D *csq1d, NcmModel *model, const gdouble t, const gdouble k)
 {
   return nc_hipert_iadiab_eval_nu (NC_HIPERT_IADIAB (model), t, k);
 }
 
 static gdouble 
-_nc_hipert_adiab_eval_dlnmnu (NcmHOAA *hoaa, NcmModel *model, const gdouble t, const gdouble k)
+_nc_hipert_adiab_eval_xi (NcmCSQ1D *csq1d, NcmModel *model, const gdouble t, const gdouble k)
 {
-  return nc_hipert_iadiab_eval_dlnmnu (NC_HIPERT_IADIAB (model), t, k);
+  return nc_hipert_iadiab_eval_xi (NC_HIPERT_IADIAB (model), t, k);
 }
 
-static void 
-_nc_hipert_adiab_eval_system (NcmHOAA *hoaa, NcmModel *model, const gdouble t, const gdouble k, gdouble *nu, gdouble *dlnmnu, gdouble *Vnu)
+static gdouble
+_nc_hipert_adiab_eval_F1 (NcmCSQ1D *csq1d, NcmModel *model, const gdouble t, const gdouble k)
 {
-  Vnu[0] = 0.0;  
-  nc_hipert_iadiab_eval_system (NC_HIPERT_IADIAB (model), t, k, nu, dlnmnu);
+  return nc_hipert_iadiab_eval_F1 (NC_HIPERT_IADIAB (model), t, k);
 }
 
-static void 
-_nc_hipert_adiab_prepare (NcmHOAA *hoaa, NcmModel *model)
+static gdouble
+_nc_hipert_adiab_eval_F2 (NcmCSQ1D *csq1d, NcmModel *model, const gdouble t, const gdouble k)
 {
-  g_assert (NC_IS_HIPERT_IADIAB (model));
+  return nc_hipert_iadiab_eval_F2 (NC_HIPERT_IADIAB (model), t, k);
 }
 
-static guint 
-_nc_hipert_adiab_nsing (NcmHOAA *hoaa, NcmModel *model, const gdouble k)
-{
-  return nc_hipert_iadiab_nsing (NC_HIPERT_IADIAB (model), k);
-}
 
 static void 
-_nc_hipert_adiab_get_sing_info (NcmHOAA *hoaa, NcmModel *model, const gdouble k, const guint sing, gdouble *ts, gdouble *dts_i, gdouble *dts_f, NcmHOAASingType *st)
-{
-  nc_hipert_iadiab_get_sing_info (NC_HIPERT_IADIAB (model), k, sing, ts, dts_i, dts_f, st);
-}
-
-static gdouble 
-_nc_hipert_adiab_eval_sing_mnu (NcmHOAA *hoaa, NcmModel *model, const gdouble t_m_ts, const gdouble k, const guint sing)
+_nc_hipert_adiab_eval_system (NcmCSQ1D *csq1d, NcmModel *model, const gdouble t, const gdouble k, gdouble *nu, gdouble *dlnmnu, gdouble *Vnu)
 {
-  return nc_hipert_iadiab_eval_sing_mnu (NC_HIPERT_IADIAB (model), t_m_ts, k, sing);
-}
-
-static gdouble 
-_nc_hipert_adiab_eval_sing_dlnmnu (NcmHOAA *hoaa, NcmModel *model, const gdouble t_m_ts, const gdouble k, const guint sing)
-{
-  return nc_hipert_iadiab_eval_sing_dlnmnu (NC_HIPERT_IADIAB (model), t_m_ts, k, sing);
+  Vnu[0] = 0.0;  
```

Comment:
remove (check if necessary for something)

Reply:
Removed Vnu parameter from all prototypes